life/about-me-instructions.md

```diff
@@ -0,0 +1,24 @@
+# About Me Sections
+
+I'm a big fan of self-care, and I think part of that involves recognizing that you can still learn something about yourself and acknowledging your happies and crappies. This section is specifically for those kinds of TILs.
+
+I have a separate branch on this for my own personal use that at least for the time being, I don't plan on pushing. That said, I wanted to add this because I'm getting a lot out of it and wanted to share the idea.
```

Comment:
yea my branch is called laurpaik I'm pretty sure. My only thing is I'm not sure if I want to merge to master because a lot of it's reflection, so unless you or other future contributors want to merge, does it need its own directory? I mean if you or anyone else merge I probably will too, but since I'm the only one who has one it feels kind of weird.